main_pprof.go

```diff
@@ -0,0 +1,54 @@
+//go:build !no_pprof
+// +build !no_pprof
+
+package main
+
+import (
+	"flag"
+	"os"
+	"runtime/pprof"
+
+	"fortio.org/log"
+)
+
+var (
+	cpuprofile = flag.String("profile-cpu", "", "write cpu profile to `file`")
+	memprofile = flag.String("profile-mem", "", "write memory profile to `file`")
+)
+
+func init() {
+	hookBefore = pprofBeforeHook
+	hookAfter = pprofAfterHook
```

Comment:
hopefully that's ok in this case @ccoVeille (to have with and without pprof version without duplicating the whole main and without needing a main_no_pprof.go test

Reply:
This one is OK 👍